static/js/services/enketo.js

```diff
@@ -343,7 +342,9 @@ angular.module('inboxServices').service('Enketo',
         })
         .then(transformXml)
         .then(function(doc) {
-          var form = renderFromXmls(doc, selector, instanceData);
+          return renderFromXmls(doc, selector, instanceData);
+        })
+        .then(function(form) {
```

Comment:
For my understanding: this doesn't actually change anything, does it?  (except add more promisey steps)

Reply:
As above.